dlk/python/dlk/templates/src/func/arch/aarch64_neon/quantized_conv2d.cpp

```diff
@@ -62,14 +62,15 @@ void QuantizedConv2D(QUANTIZED_NOT_PACKED input[], T_UINT kernel[],
   else
     std::memset((void *)p.device_output_buf, 0, size * sizeof(BIN_CONV_OUTPUT));
 
-  if ((kh == 3 && kw == 3 && padding == 1) ||
-      (kh == 1 && kw == 1 && padding == 0)) {
+  if (kh == 3 && kw == 3 && padding == 1) {
     if ((ic % TilingInTypeBitWidth) == 0) {
-#if defined(USE_NEON) && !defined(RUN_ON_FPGA)
       dlk::impl::QuantizedConv2DTiling(input, kernel, p);
-#else
+    } else {
       dlk::impl::QuantizedConv2DKn2Row(input, kernel, p);
-#endif
+    }
+  } else if (kh == 1 && kw == 1 && padding == 0) {
+    if ((ic % TilingInTypeBitWidth) == 0) {
```

Comment:
Have a strong filling that `QuantizedConv2DKn2Row()` also will not work properly for case `ic % TilingInTypeBitWidth != 0`... :thinking:

Reply:
This is original implementation. Nikolay-san and I are seeing this issue.